preprocessing/granular_intent.py

```diff
@@ -0,0 +1,101 @@
+            "GET_ESTIMATED_ARRIVAL": "get_time_estimate",
+            "GET_ESTIMATED_DEPARTURE": "get_time_estimate",
+            "GET_ESTIMATED_DURATION": "get_time_estimate",
+            "GET_EVENT": "get_event",
+            "GET_EVENT_ATTENDEE": "get_event",
+            "GET_EVENT_ATTENDEE_AMOUNT": "get_event",
+            "GET_EVENT_ORGANIZER": "get_event",
+            "GET_INFO_ROAD_CONDITION": "get_road_info",
+            "GET_INFO_ROUTE": "get_road_info",
+            "GET_INFO_TRAFFIC": "get_road_info",
+            "GET_LOCATION": "get_location",
+            "GET_LOCATION_HOME": "get_location",
+            "GET_LOCATION_HOMETOWN": "get_location",
+            "GET_LOCATION_SCHOOL": "get_location",
+            "GET_LOCATION_WORK": "get_location",
+            "NEGATION": "unsupported",
+            "UNINTELLIGIBLE": "unsupported",
+            "UNSUPPORTED": "unsupported",
+            "UNSUPPORTED_EVENT": "unsupported",
+            "UNSUPPORTED_NAVIGATION": "unsupported",
+            "UPDATE_DIRECTIONS": "update_directions"
+            },
+        "original":{
+            "COMBINE": "combine",
+            "GET_CONTACT": "get_contact",
+            "GET_DIRECTIONS": "get_directions",
+            "GET_DISTANCE": "get_distance",
+            "GET_ESTIMATED_ARRIVAL": "get_estimated_arrival",
+            "GET_ESTIMATED_DEPARTURE": "get_estimated_departure",
+            "GET_ESTIMATED_DURATION": "get_estimated_duration",
+            "GET_EVENT": "get_event",
+            "GET_EVENT_ATTENDEE": "get_event_attendee",
+            "GET_EVENT_ATTENDEE_AMOUNT": "get_event_attendee_amount",
+            "GET_EVENT_ORGANIZER": "get_event_organizer",
+            "GET_INFO_ROAD_CONDITION": "get_info_road_condition",
+            "GET_INFO_ROUTE": "get_info_route",
+            "GET_INFO_TRAFFIC": "get_info_traffic",
+            "GET_LOCATION": "get_location",
+            "GET_LOCATION_HOME": "get_location_home",
+            "GET_LOCATION_HOMETOWN": "get_location_hometown",
+            "GET_LOCATION_SCHOOL": "get_location_school",
+            "GET_LOCATION_WORK": "get_location_work",
+            "NEGATION": "negation",
+            "UNINTELLIGIBLE": "unintelligible",
+            "UNSUPPORTED": "unsupported",
+            "UNSUPPORTED_EVENT": "unsupported_event",
+            "UNSUPPORTED_NAVIGATION": "unsupported_navigation",
+            "UPDATE_DIRECTIONS": "update_directions"
+            }
+}
+#  intent_list = ['NEGATION', 'GET_LOCATION_WORK', 'GET_INFO_ROAD_CONDITION', 'UNSUPPORTED_NAVIGATION', 'GET_CONTACT', 'GET_INFO_ROUTE', 'GET_LOCATION_SCHOOL', 'GET_DIRECTIONS', 'GET_ESTIMATED_DURATION', 'GET_INFO_TRAFFIC', 'GET_ESTIMATED_ARRIVAL', 'GET_EVENT', 'GET_ESTIMATED_DEPARTURE', 'UNSUPPORTED_EVENT', 'GET_LOCATION', 'UNINTELLIGIBLE', 'COMBINE', 'GET_EVENT_ORGANIZER', 'GET_EVENT_ATTENDEE', 'GET_LOCATION_HOME', 'GET_DISTANCE', 'GET_EVENT_ATTENDEE_AMOUNT', 'GET_LOCATION_HOMETOWN', 'UPDATE_DIRECTIONS', 'UNSUPPORTED']
```

Comment:
Remove?

Reply:
Removed!

Reply:
Same for this change, I don't see your updated file.

Reply:
I see it now!